ruleops-basic/pom.xml

```diff
@@ -35,6 +21,12 @@
     </dependencies>
   </dependencyManagement>
   <dependencies>
+    <dependency>
+      <groupId>org.drools.ruleops</groupId>
+      <artifactId>ruleops-core</artifactId>
+      <version>1.0.0-SNAPSHOT</version>
```

Comment:
Same here, we should use a property for the version

Reply:
Same suggestion as above?